CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,188 @@
+# RAI Contributors Code of Conduct
+
+RAI adopts the [ROS Code of Conduct](https://osrf.github.io/code_of_conduct/) maintained by the OSRF.
+We expand the scope of the Code of Conduct to all official RAI communication channels, including, but not limited to:
+
+- RAI source code repositories
+
+What follows is the adopted ROS Code of Conduct (retrieved: 2024-08-22). It also includes contact information to RAI internal Conduct Team.
+
+# ROS Code of Conduct
+
+## Code of Conduct for ROS
+
+[View the Project on GitHub](https://github.com/osrf/code_of_conduct)
```

Comment:
```suggestion
[View the Project on GitHub](https://osrf.github.io/code_of_conduct/)
```

Reply:
That is correct, but this is copied directly, so I see no reason to modify it.